docs/triggerbindings.md

```diff
@@ -21,6 +21,8 @@ spec:
           value: $(event.head_commit.id)
         - name: gitrepositoryurl
           value: $(event.repository.url)
```

Comment:
Would you add the reference of the syntax of the path string?  It seems supporting this https://github.com/tidwall/gjson/blob/master/SYNTAX.md

Reply:
I think this additional functionality is a good thing to keep in mind as we move forward. However, at the moment, I don't think we should officially "support" it. This is a first iteration, so I think it would be good to keep things simple for now. Plus, we don't want to grow too dependent on this library while we're still early on in the development process.